### DISCUSSION TITLE
[Feature Request] [combobox] Enhance Multiselect selection display

### DISCUSSION THREAD

**Discussion by erikkroes** [ing-bank/lion#954] [General]
## Expected behavior

When using a screen reader and focussing on a combobox with `multiple-choice`, all selected values are read, visually indicated and can be changed.

## Actual Behavior

Only the "typed" text is being read. Not the already (pre)selected options. This makes it unclear what is there, including what can be deleted (by keyboard only?) or what can still be added.
Are these values also meant to be unique, or can one option be picked multiple times? It's not clear from the listbox. Previously selected values get highlighted there.

<img width="1049" alt="" src="https://user-images.githubusercontent.com/10405024/94410236-6d372a80-0177-11eb-8b94-a98f1ce89e96.png">
<img width="1028" alt="" src="https://user-images.githubusercontent.com/10405024/94410239-6e685780-0177-11eb-8ea0-c25d066f5e88.png">

I would slim down this component and remove the `multiple-choice` until there is a clear plan on how to handle these chips.

**Comment by tlouisse** [comment]
The example states that the selection-display is an example implementation, and therefore the selection-display is only put in the docs folder.
I will add a warning and make it clearer in the docs that it will be upon the Sub classer for now to handle a11y 👍 

Will keep this issue as a feature request.

**Comment by tlouisse** [comment]
More valuable input for the multiselection display:
- https://github.com/ing-bank/lion/issues/1004
- https://github.com/ing-bank/lion/issues/1003
